bool is_new_episode
int16 start_point
